mysql-test/suite/innodb/t/doublewrite_debug.test

```diff
@@ -145,7 +156,7 @@ WHERE engine = 'innodb'
 AND support IN ('YES', 'DEFAULT', 'ENABLED');
 
 --let $ibp=--innodb-log-group-home-dir=$bugdir --innodb-data-home-dir=$bugdir
---let $ibd=$ibp --innodb-undo-tablespaces=0
+--let $ibp=$ibp --innodb-undo-tablespaces=0
```

Comment:
On merge to 11.0, we must revert the change to `doublewrite_debug.combinations` that had been made in the previous merge 8c8bce05d22e0e97e844fb2518e23aa04a544cd9.